modules/javafx.graphics/src/main/java/com/sun/javafx/image/PixelUtils.java

```diff
@@ -207,7 +212,21 @@ PixelConverter<T, U> getConverter(PixelGetter<T> src, PixelSetter<U> dst)
     public static ByteToBytePixelConverter
         getB2BConverter(PixelGetter<ByteBuffer> src, PixelSetter<ByteBuffer> dst)
     {
-        if (src ==        ByteBgra.getter) {
+        if (src ==        ByteRgba.getter) {
```

Comment:
The code style in this area is very... special. However, I'm hesitant to do a wholesale reformatting here. This will pollute the Git change log with not all that much benefits.